scripts/download_chrome.js

```diff
@@ -25,6 +26,9 @@ const outputPath = path.resolve(process.cwd(), 'dist', process.argv[3] || `plugi
 
 const browserVersion = Browser.CHROME;
 
+// Clean up download folder if exists
+childProcess.execFileSync('rm', ['-rf', `${outputPath}/chrome`]);
+
 async function download() {
```

Comment:
```js
async function download() {
    const buildId = await resolveBuildId(browserVersion, platform, 'latest');
    console.log(`Installing ${browserVersion} into ${outputPath}`);
    const executablePath = computeExecutablePath({buildId, browser: browserVersion, platform, cacheDir: ''});
    const browser = await install({
        cacheDir: outputPath,
        browser: browserVersion,
        platform,
        buildId,
    });

    return {browser, executablePath};
}

download().then(({executablePath}) => {
    console.log(`${browserVersion} downloaded into:`, outputPath);

    // All the chrome files should be in the plugin 'chrome' folder
    // This is used to set the puppeteer.executablePath
    const out = executablePath.substring(0, executablePath.lastIndexOf("/"));
    const chromeBinDir = `${outputPath}/${out}`
    console.log(`Moving ${chromeBinDir} content into ${outputPath}/chrome`);
    childProcess.execFileSync('mv', [chromeBinDir, `${outputPath}/tmp`]);
    childProcess.execFileSync('rm', ['-r', `${outputPath}/chrome`]);
    childProcess.execFileSync('mv', [`${outputPath}/tmp`, `${outputPath}/chrome`]);
});
```

This should work independent of the OS and move the chrome executable directly to the `chrome` folder. The path also needs to be adjusted in `app.ts`. However, this doesn't fix the plugin startup error for me, not sure if it'd be different on Windows.

Reply:
I think this doesn't work on Mac because Chrome executable is not at the root of the Chrome folder as on Linux or Windows. 

I'll look into another solution. I think if we can pass the `buildId` to the package, then we can use the `computeExecutableFunction` to get the whole executable path.